cli/cmd/common.go

```diff
@@ -23,10 +23,15 @@ var policyPathFlag string
 var validateFlag bool = false
 var previewFlag bool = false
 var showStagesFlag bool = false
+var nonInteractiveFlag bool = false
 var formatFlag string
 
 var supportedExt = []string{"*.yaml", "*.yml", "*.jinja"}
 
+func initNonInteractiveFlag(command *cobra.Command) {
```

Comment:
We should simply add this on the root cli command (it should be consistent across *all* cli commands)